### ISSUE TITLE
Error when creating a tagged event

### ISSUE THREAD

**Issue by pgramme** [philippejadin/agorakit#238]
Hi !

I encountered a non-blocking bug when creating an event with a tag on agora.reseautransition.be (sorry, don't know what version is used):

```
A fatal error occured
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'taggable_id' cannot be null (SQL: insert into `taggable_taggables` (`created_at`, `tag_id`, `taggable_id`, `taggable_type`, `updated_at`) values (2019-06-03 18:27:00, 547, , App\Action, 2019-06-03 18:27:00))

Contact the server admin to get help and explain what you were trying to do. Provide this error message to help fix the bug.
```

This is non-blocking because I can create the event without the tag and then add the tag right afterwards.

I think the bug was introduced since the latest update, but not sure I had already tested that scenario before.

And most of all, thanks for the great work.

**Comment by philippejadin** [comment]
There is definitely a bug that must be fixed. Thank you for reporting!

**Comment by philippejadin** [comment]
solved by 8310aabb82a0a090d8e838267e99c32aee1d3434

**Comment by pgramme** [comment]
I'm really late, but thanks for the fast fix !
